test/jasmine/bundle_tests/bar_test.js

```diff
@@ -1,16 +1,15 @@
-var d3SelectAll = require('../../strict-d3').selectAll;
-
 var Plotly = require('@lib/core');
-var PlotlyBar = require('@lib/bar');
+var Scatter = require('@lib/scatter');
+var Bar = require('@lib/bar');
 
+var d3SelectAll = require('../../strict-d3').selectAll;
 var createGraphDiv = require('../assets/create_graph_div');
 var destroyGraphDiv = require('../assets/destroy_graph_div');
 
-
 describe('Bundle with bar', function() {
     'use strict';
 
-    Plotly.register(PlotlyBar);
+    Plotly.register([Scatter, Bar]);
```

Comment:
Can we get away with NOT registering `scatter` then, if we remove the scatter trace from the mock? Similarly for the other bundle tests, if we're going to make `scatter` optional, we should remove it in any bundle test that's not specifically calling for it.

Reply:
Yes we can.
But I thought we could try to address that part in a separate PR.

Reply:
I don't think that should be a separate PR - either it works and custom bundles can drop cartesian entirely and we test this capability, or it doesn't work and we shouldn't continue with this PR.

Reply:
OK. I adjusted the tests in https://github.com/plotly/plotly.js/pull/5535/commits/4529cc1a872278eb5a4e7dabeef914a9077d5c4e.
Next stop would be handling default trace type...